example/src/Main.hx

```diff
@@ -62,33 +66,35 @@ class Main
 			.setTextAreaRows("file info", 10)
 			.addImage("image", "");
 
-		// parse example
-		var json ='{
-			"title": "Live Parse Demo",
-			"x": 490,
-			"y": 350,
-			"draggable": true,
-			"collapsible": true,
-			"snapToGrid": true,
-			"gridSize": 10,
-			"controls": [{
-				"type": "range",
-				"title": "range test",
-				"min": 0,
-				"max": 100,
-				"value": 50,
-				"step": 1
-			},{
-				"type": "textarea",
-				"title": "text area test",
-				"value": "just some text"
-			},{
-				"type": "boolean",
-				"title": "boolean test",
-				"value": true
-			}]
-		}';
-		var panel6 = QuickSettings.parse(json);
+
```

Comment:
I think you can just remove the old stuff, no need keep it in comments (That's why we use source control 😄)